src/tests/functional/plugin/shared/src/behavior/ov_infer_request/io_tensor.cpp

```diff
@@ -46,29 +45,35 @@ TEST_P(OVInferRequestIOTensorTest, failToSetNullptrForOutput) {
     ASSERT_THROW(req.set_tensor(output, {}), ov::Exception);
 }
 
-TEST_P(OVInferRequestIOTensorTest, getAfterSetInputDoNotChangeInput) {
-    auto tensor = utils::create_and_fill_tensor(input.get_element_type(), input.get_shape());
+TEST_P(OVInferRequestIOTensorTest, canSetAndGetInput) {
+    auto shape = input.get_shape();
+    shape[0] = 2;
+    shape[1] = static_cast<int>(shape[1] / 2);
```

Comment:
```suggestion
    shape[1] = static_cast<size_t>(shape[1] / 2);
```
or 
```suggestion
    shape[1] /= shape[0];
```
`ov::Shape` value type is `size_t`, is cast required at all?

Check all similar occurrences in this PR.

Reply:
thank you, changed